architecture-decision-records/0021-service-unavailable-page.md

```diff
@@ -0,0 +1,37 @@
+# 20. Deploying a service unavailable page 
+
+**Date:** 2021-07-14
+
+## Context
+We need a mechanism to manually deploy a service unavailable page. There are two main reasons we might want to do this. 
+* There is a serious issue with the site that we can't fix immediately.
+* We want to carry out maintenance which will affect service. 
```

Comment:
Should read: "... which will affect **the** service."

Reply:
I've changed it :+1: